pkg/virtual/initializingworkspaces/builder/build.go

```diff
@@ -81,9 +79,24 @@ func BuildVirtualWorkspace(
 		return info.IsResourceRequest && info.APIGroup == tenancyv1alpha1.SchemeGroupVersion.Group && info.Resource == "clusterworkspaces"
 	}
 
-	filterReadyCh := make(chan struct{})
-	filterName := initializingworkspaces.VirtualWorkspaceName + "-filtering"
-	filtering := &virtualworkspacesdynamic.DynamicVirtualWorkspace{
+	clusterWorkspaceResource := apisv1alpha1.APIResourceSchema{}
+	if err := rootphase0.Unmarshal("apiresourceschema-clusterworkspaces.tenancy.kcp.dev.yaml", &clusterWorkspaceResource); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal clusterworkspace resource: %w", err)
+	}
+	bs, err := json.Marshal(&v1.JSONSchemaProps{
+		Type:                   "object",
+		XPreserveUnknownFields: pointer.BoolPtr(true),
+	})
+	if err != nil {
+		return nil, err
+	}
+	for i := range clusterWorkspaceResource.Spec.Versions {
+		v := &clusterWorkspaceResource.Spec.Versions[i]
+		v.Schema.Raw = bs // wipe schemas. We don't want validation here.
+	}
+
+	wildcardWorkspacesName := initializingworkspaces.VirtualWorkspaceName + "-wildcard-workspaces"
```

Comment:
why rename?

Reply:
to make it understandable without reading deep code